README.md

```diff
@@ -59,44 +59,54 @@ It's 100% Open Source and licensed under the [APACHE2](LICENSE).
 
 ## Usage
 
+
+### MFA Providers
+
+This documentation only recognises Google Authenticator and Duo.
+
+- Google Authenticator
+  - Google Authenticator is a free, open-source, and easy to setup MFA.
+  - It's less secure than Duo because tokens are stored on the server under each user account.
+- Duo
+  - Duo is an enterprise paid-for service. It has excellent push-notification support so that MFA requests can be approved or declined without opening an app.
```

Comment:
```suggestion
  - Duo is a paid enterprise service. It has excellent mobile push-notification support so that MFA requests can be approved or declined without opening an authenticator app.
```